build/common.sh

```diff
@@ -502,6 +502,7 @@ function kube::build::ensure_data_container() {
       --volume /usr/local/go/pkg/darwin_386_cgo
       --volume /usr/local/go/pkg/windows_amd64_cgo
       --volume /usr/local/go/pkg/windows_386_cgo
+      --volume /godeps
```

Comment:
any action needed like push a new image? Will that work with old images laying around on the developer's machines?